backend/src/src-cvesearch/src/main/java/com/bosch/osmi/sw360/cvesearch/helper/VulnerabilityUtils.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2016.
+ * Part of the SW360 Portal Project.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package com.bosch.osmi.sw360.cvesearch.helper;
+
+import com.siemens.sw360.datahandler.common.CommonUtils;
+import com.siemens.sw360.datahandler.thrift.RequestStatus;
+import com.siemens.sw360.datahandler.thrift.cvesearch.UpdateType;
+import com.siemens.sw360.datahandler.thrift.cvesearch.VulnerabilityUpdateStatus;
+import com.siemens.sw360.datahandler.thrift.vulnerabilities.Vulnerability;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static com.siemens.sw360.datahandler.common.CommonUtils.nullToEmptyList;
+
+public class VulnerabilityUtils {
+
+    public static VulnerabilityUpdateStatus getEmptyVulnerabilityUpdateStatus(RequestStatus requestStatus){
```

Comment:
would getEmptsVulnerabilityStatus(){return getEmptsVulnerabilityStatus(RequestStatus.Success) } not do the tirck?